src/hooks/useNavigation.js

```diff
@@ -20,7 +20,7 @@ export const useNavigation = defaultSection => {
 const handleKeyPress = (event, focusables, defaultSection) => {
   const { key } = event;
 
-  if (NON_NAVIGATION_KEYS.includes(key)) return;
+  if (NON_NAVIGATION_KEYS.includes(key) || event.metaKey) return;
```

Comment:
Works well!

I wonder if we should also focus on the last focusable element when a user presses meta + down? same with meta + up.

The problem with not doing it is that when pressing meta + down, you get scrolled down, and then on the first arrow keypress, you get sent back to the top.

Something like that would fix the issue:
```js
const handleKeyPress = (event, focusables, defaultSection) => {
  const { key, metaKey } = event;

  if (NON_NAVIGATION_KEYS.includes(key)) return;

  if (
    [
      KEYS.ARROW_UP,
      KEYS.ARROW_RIGHT,
      KEYS.ARROW_DOWN,
      KEYS.ARROW_LEFT,
    ].includes(key) &&
    !metaKey
  ) {
    event.preventDefault();
  }

  const { activeElement } = document;

  const currentTabIndex = Array.prototype.indexOf.call(
    focusables,
    activeElement,
  );

  const { section, layout } = activeElement.dataset;

  let nextTabIndex;
  if (metaKey) {
    if ([KEYS.UP, KEYS.ARROW_UP].includes(key)) {
      nextTabIndex = 0;
    } else if ([KEYS.DOWN, KEYS.ARROW_DOWN].includes(key)) {
      nextTabIndex = focusables.length - 1;
    }
  } else if (key === KEYS.TOP) {
...
...
```

Reply:
What do you think?

Reply:
Sounds like a good idea, I pushed the changes now